config/initializers/docker.rb

```diff
@@ -2,8 +2,8 @@
 if Rails.env.development? && File.file?("/.dockerenv")
   # Using shell tools so we don't need to require Socket and IPAddr
   host_ip = `ip route|awk '/default/ { print $3 }'`.strip
-  logger = Logger.new($stdout)
-  logger.info "Allowing #{host_ip} for BetterErrors and Web Console"
+  # logger = Logger.new($stdout)
+  # logger.info "Allowing #{host_ip} for BetterErrors and Web Console"
```

Comment:
Commented code should either be uncommented or removed IMO, we have the git history if we ever need it again.

Reply:
removing it!